Makefile

```diff
@@ -16,10 +16,11 @@ DIST := $(BASE_PATH)/dist
 default: build
 
 .PHONY: GIT-VERSION-FILE
-GIT-VERSION-FILE: 
+GIT-VERSION-FILE:
 	@sh ./GIT-VERSION-GEN
 -include GIT-VERSION-FILE
 
+GIT_DESC := $(shell git describe --tags --always --dirty --match "v[0-9]*")
```

Comment:
This will break a local `make docker` build unless you also remove `.git` from top-level `.dockerignore`

Point of the GIT-VERSION-FILE was to avoid copying .git into the docker container to speed up build.

Maybe change to

```
GIT_DESC ?= $(shell git describe ...)
```

Reply:
Now that I understand the include, I changes the github workflow to run the `GIT-VERSION-GEN` script.